### PR TITLE
New package: editorconfig-core-lua-0.2.0

### PR DESCRIPTION
**Pull request by NoctuaNivalis**
For use in vis and other editors using lua.

### CODE REVIEW

File: srcpkgs/editorconfig-core-lua/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'editorconfig-lua'
+pkgname=editorconfig-core-lua
+version=0.2.0
+revision=1
+build_style=cmake
+hostmakedepends=""
```
Comment (maxice8) [inline srcpkgs/editorconfig-core-lua/template]: empty variable

File: srcpkgs/editorconfig-core-lua/template [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+makedepends="editorconfig-devel lua-devel"
+depends="lua"
```
Comment (maxice8) [inline srcpkgs/editorconfig-core-lua/template]: lua is already found by shlibs

```
=> editorconfig-core-lua-0.2.0_1: running pre-pkg hook: 04-generate-runtime-deps ...
   SONAME: libeditorconfig.so.0 <-> editorconfig>=0.12.2_1
   SONAME: liblua5.3.so.5.3 <-> lua>=5.3.1_2
   SONAME: libc.so.6 <-> glibc>=2.28_1
```

Comment (NoctuaNivalis) [reply srcpkgs/editorconfig-core-lua/template]: Cool, didn't know about this automatic detection, BRB

Comment (maxice8) [reply srcpkgs/editorconfig-core-lua/template]: Yes, it is documented on https://github.com/void-linux/void-packages/blob/master/Manual.md#runtime-dependencies

**Pull request merged**